index.html

```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
         <div class="image">
             <div class="bg-image">
                 <h1>Hilltop Farm</h1>
-                <h3>Organic farm in Ghana</h3>
+                <h3>The best among all organic farms in Ghana</h3>
+                 <div class="blue-btn"> <!--Get Started button -->
+                    <a class="first-link" href="">
+                      Get Started
```

Comment:
Use only one CTA button. Not necessary to use get started and learn more to confude your users.

If you wish to add another CTA button for a purpose, show both buttons then.